10up/simple-podcasting
# Feature/issue 104 remove file without removing block

### Description of the Change

Adds a remove button to the toolbar so that the editor has an alternate means to remove the audio and replace it.

### Alternate Designs

None.

### Benefits

Editors will have two ways to remove/replace files in the block.

### Possible Drawbacks

Most core blocks don't have this particular UI where the editor has two options to do the same process. It may be confusing.

### Verification Process

1. Add block with audio file
2. Verify information changed in the block
3. Remove the audio using the new button
4. Verify information changed in the block

### Checklist:






### Applicable Issues

https://github.com/10up/simple-podcasting/issues/104

### Changelog Entry


Added ability for editors to remove the current audio file from the block without deleting the block in an alternative way.
